tests/flow/test_list.py

```diff
@@ -143,3 +143,633 @@ def test03_null_list_function_inputs(self):
+        expected_result = []
+        actual_result = redis_graph.query(query)
+        self.env.assertEquals(actual_result.result_set, expected_result)
+
+        query = """MATCH (n:X) RETURN toFloatList([n])"""
+        expected_result = []
+        actual_result = redis_graph.query(query)
+        self.env.assertEquals(actual_result.result_set, expected_result)
+
+        query = """MATCH ()-[e]->() RETURN toFloatList(e)"""
+        expected_result = []
+        actual_result = redis_graph.query(query)
+        self.env.assertEquals(actual_result.result_set, expected_result)
+
+        query = """MATCH ()-[e]->() RETURN toFloatList([e])"""
+        expected_result = []
+        actual_result = redis_graph.query(query)
+        self.env.assertEquals(actual_result.result_set, expected_result)
+
+        query = """MATCH p=()-[]->() RETURN toFloatList(p)"""
+        expected_result = []
+        actual_result = redis_graph.query(query)
+        self.env.assertEquals(actual_result.result_set, expected_result)
+
+        query = """MATCH p=()-[]->() RETURN toFloatList([p])"""
+        expected_result = []
+        actual_result = redis_graph.query(query)
+        self.env.assertEquals(actual_result.result_set, expected_result)
+
+        try:
+            redis_graph.query("RETURN toFloatList(true)")
+            self.env.assertTrue(False)
+        except ResponseError as e:
+            self.env.assertContains("Type mismatch: expected List but was Boolean", str(e))
+
+        try:
+            redis_graph.query("RETURN toFloatList(7.05)")
+            self.env.assertTrue(False)
+        except ResponseError as e:
+            self.env.assertContains("Type mismatch: expected List but was Float", str(e))
+
+        try:
+            redis_graph.query("RETURN toFloatList(7)")
+            self.env.assertTrue(False)
+        except ResponseError as e:
+            self.env.assertContains("Type mismatch: expected List but was Integer", str(e))
+
+        try:
+            redis_graph.query("RETURN toFloatList('abc')")
+            self.env.assertTrue(False)
+        except ResponseError as e:
+            self.env.assertContains("Type mismatch: expected List but was String", str(e))
+
+        try:
+            query = """CREATE (a:X)"""
+            redis_graph.query(query)
+            query = """MATCH (n) RETURN toFloatList(n)"""
+            actual_result = redis_graph.query(query)
+            self.env.assertTrue(False)
+        except ResponseError as e:
+            self.env.assertContains("Type mismatch: expected List but was Node", str(e))
+            query = """MATCH (a:X) DELETE a"""
+            redis_graph.query(query)
+
+        try:
+            query = """CREATE (a:X), (b:Y)"""
+            redis_graph.query(query)
+            query = """MATCH (a:X), (b:Y) create (a)-[r:R]->(b)"""
+            redis_graph.query(query)
+            query = """MATCH ()-[e]->() RETURN toFloatList(e)"""
```

Comment:
Similar comments as for the previous test function

Reply:
Done in [b36f30d](https://github.com/RedisGraph/RedisGraph/pull/2605/commits/b36f30dfbf0af4709ba9aa7c5ba4b01be17b10fd)